### ISSUE TITLE
Contrast on on orange buttons fails accessibility

### ISSUE THREAD

**Issue by emmalearnscode** [max-programming/hacktoberfest-projects#21]
Color contrast should be at least 4.5 : 1 between background and foreground colors. Orange and offwhite is too low and therefore inaccessible to many people with visual impairments. Orange and black would pass.

**Comment by max-programming** [comment]
Thank you for making an issue and helping improve the project.  Are you talking about contrast on these buttons? If so, and you want to change it you can
![image](https://user-images.githubusercontent.com/51731966/135701654-9471edae-fb02-4c1c-adfb-002979ad3801.png)

**Comment by emmalearnscode** [comment]
Yes, those are the buttons. I can definitely have a go at changing it for you if you’re happy to assign it to me.

**Comment by janaSunrise** [comment]
We surely are open to any helpful changes! You can take this over.

**Comment by shubhraagarwal** [comment]
is this issue up for grabs??

**Comment by emmalearnscode** [comment]
> is this issue up for grabs??

No, I’m currently working on it.
